frontend/src/routes/+layout.svelte

```diff
@@ -19,12 +19,11 @@
 
 <Navbar>
 	<NavBrand href={base}>
-		<img src={logo_light} class="block h-16 dark:hidden" alt="MONDEY Logo" />
-		<img src={logo_dark} class="hidden h-16 dark:block" alt="MONDEY Logo" />
+		<img src={logo_light} class="mt-6 block h-12 dark:hidden" alt="MONDEY Logo" />
+		<img src={logo_dark} class="mt-6 hidden h-12 dark:block" alt="MONDEY Logo" />
 	</NavBrand>
 	<NavHamburger />
-	<NavUl ulClass="flex space-x-4 text-lg ">
-		<NavLi href={base} active={true}>Home</NavLi>
+	<NavUl ulClass="flex items-center mt-8 space-x-14 text-lg ">
```

Comment:
Have you checked what this looks like on a mobile device screen size?
It might make sense to only have these additional margins and x spacing on large screens - and this is easy to do as tailwind is [mobile first](https://tailwindcss.com/docs/responsive-design), so you can do something like e.g. `mt-2 md:mt-8` to have a smaller margin by default (i.e. on mobile devices), then apply the larger margin for the `md` breakpoint (>768px width)

Reply:
Thanks! I've just fixed that. Now it looks better with all screen sizes.